meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -17,6 +17,7 @@ SRCREV = "c216955d16ca275159891cad296217094d972390"
 
 SRC_URI += "file://amb_allow_sessionbus.patch \
             file://ambd.service \
+            file://ambd_fsa.service \
```

Comment:
Some options:
For sure, I need to use automobile message borker.
Then, 
1. I use the one in the sytem dbus WHERE I need to add my configuration file (where someone that use amb may brake something fro him)
2. Or I use the one in the session dbus with my configuration file in the session where no one use it.